docs/Encoders/svt-av1.md

```diff
@@ -1 +1,118 @@
- 
+# SVT-AV1
+
+This will be a quick guide for setting options svt-av1 when using with Av1an. For more complete documentation, you should read the official documentation:
+
+- [SVT-AV1 Docs - GitLab](https://gitlab.com/AOMediaCodec/SVT-AV1/-/tree/master/Docs)
+
+Make sure your svt-av1 encoder is up-to-date.
+
+# Rate control
+
+## --rc
+
+The `--rc` option selects what rate control strategy you want to use.
+
+- `--rc 0` - Constant rate factor
+- `--rc 1` - Variable bit rate
+- `--rc 2` - Constant bit rate
+
+## Constant rate factor 
+
+```
+--rc 0 --crf
+```
+
+Constant rate factor, a.k.a. constant quality. This is the most common way of determining video quality. You set --crf to be a number between 1-63, and the encoder will work out a bit-rate to keep constant quality. The lower the number, the less compression, the higher it is, the more compression will be used. Anything lower than 20 is considered "hi-fi", 30+ is for mini encodes.
+
+**Example:**
+
+```
+av1an ... -v " --rc 0 --crf 24 --preset 4 --input-depth 10 --tune 0" ...
+```
+
+## Variable bit-rate
+
+```
+--rc 1 --tbr
+```
+
+Variable bit-rate. Requires you to set target bit-rate `--tbr`. Recommended to use 2 passes for better quality as well, make sure to set two pass in Av1an, not SVT!
```

Comment:
I don't think 2-pass VBR currently works properly with av1an, given the way chunking works. You'll end up with each chunk averaging your target bitrate rather than the whole video averaging your target bitrate.

Reply:
I do know 2-pass is possible in aom. Do we need to set 2 pass in the svt options instead?

Reply:
It "works" but it doesn't work the way you expect (with aom or svt). I think a (terrible) drawing might be the easiest way to explain it.

Running 2-pass VBR in aomenc (or any encoder) directly, you would see a bitrate distribution like this (red = keyframes):

![expected-vbr](https://user-images.githubusercontent.com/5951392/163744739-2d1971a6-7495-4698-af6a-35d79ec92e04.png)


However, because of how chunking works in av1an, attempting to do 2-pass VBR in av1an will get you something like this:

![av1an-vbr](https://user-images.githubusercontent.com/5951392/163744741-b7ce2361-4260-4aff-a51d-f4a4bbf9ba70.png)

Which is very different than we expect. Because each chunk averages to our requested bitrate, it means that chunks which need more bitrate will be starved of it, and chunks which need less bitrate will be bloated.

Reply:
Thanks for the info. I have removed the advice for the time being.